src/main/java/core/basesyntax/service/impl/FruitMapperImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.FruitMapper;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitMapperImpl implements FruitMapper {
+
+    @Override
+    public List<FruitTransaction> mapData(List<String> list) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        list.remove(0);
+        for (String item : list) {
+            String[] data = item.split(",");
+            String operation = data[0];
+            String fruitType = data[1];
+            int quantity = Integer.parseInt(data[2]);
```

Comment:
What do you think about making them constant? We can give some meaningful names, like `OPERATION_INDEX`, `FRUIT_NAME_INDEX` etc., which will improve readability.